lib/formatters.js

```diff
@@ -134,14 +134,32 @@ function colorize(color, str) {
  * @deprecated
  */
 function formatTrace(level, context, message, args, err) {
-  args.unshift(
-      templateTrace + message,
-      (new Date()).toISOString(),
-      level,
-      context.corr || notAvailable,
-      context.trans || notAvailable,
-      context.op || notAvailable
-  );
+
+  var recontext = {
+    time: (new Date()).toISOString(),
+    lvl: level,
+    corr: context.corr || notAvailable,
+    trans: context.trans || notAvailable,
+    op: context.op || notAvailable
```

Comment:
`corr`, `trans`and `op` could be directly set to `notAvailable` as their default value since if they exist in `context` they will be overwritten by the `Object.keys()` code below to their `context`value.